### PR TITLE
Fix dev

### PR DESCRIPTION
**Pull request by TurtleMinecraft**

### CODE REVIEW

File: src/main/java/com/spikes2212/dashboard/Namespace.java [hunk 1]
```diff
@@ -14,58 +13,186 @@
...
+    /**
+     * Gets a string value from the namespace.
+     *
+     * @param key the key of the value
+     * @return the desired value
+     */
     String getString(String key);
 
+    /**
+     * Adds a number value to the namespace.
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/com/spikes2212/dashboard/Namespace.java:136]: number supplier

File: src/main/java/com/spikes2212/dashboard/Namespace.java [hunk 2]
```diff
@@ -14,58 +13,186 @@
...
+     * Gets a number value from the namespace.
+     *
+     * @param key the key of the value
+     * @return the desired value
+     */
     double getNumber(String key);
 
+
+    /**
+     * Adds a boolean value to the namespace.
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/com/spikes2212/dashboard/Namespace.java:163]: boolean supplier

File: src/main/java/com/spikes2212/dashboard/Namespace.java [hunk 3]
```diff
@@ -14,58 +13,186 @@
...
+     * @return the desired value
+     */
     boolean getBoolean(String key);
 
+    /**
+     * Adds a set of {@link PIDSettings} values to the namespace.
+     *
+     * @param name               the name to be given to the settings
+     * @param initialPIDSettings the PID settings to be added
+     * @return PID settings with the most recent value from the network tables
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/com/spikes2212/dashboard/Namespace.java:194]: pid settings with the values from the network tables

File: src/main/java/com/spikes2212/dashboard/Namespace.java [hunk 4]
```diff
@@ -14,58 +13,186 @@
...
+    /**
+     * Gets a boolean value from the namespace.
+     *
+     * @param key the key of the value
+     * @return the desired value
+     */
     boolean getBoolean(String key);
 
+    /**
+     * Adds a set of {@link PIDSettings} values to the namespace.
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/com/spikes2212/dashboard/Namespace.java:190]: to a designated {@link ChildNamespace}

File: src/main/java/com/spikes2212/dashboard/Namespace.java [hunk 5]
```diff
@@ -76,6 +203,13 @@ default PIDSettings addPIDNamespace(String name, PIDSettings initialPIDSettings)
         return new PIDSettings(kP, kI, kD, tolerance, waitTime);
     }
 
+    /**
+     * Adds a set of {@link FeedForwardSettings} values to the namespace.
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/com/spikes2212/dashboard/Namespace.java:207]: the same comments as on 'addPIDNamespace'

File: src/main/java/com/spikes2212/dashboard/Namespace.java [hunk 6]
```diff
@@ -14,58 +13,186 @@
...
+     * @param key the key of the value
+     * @return the desired value
+     */
     boolean getBoolean(String key);
 
+    /**
+     * Adds a set of {@link PIDSettings} values to the namespace.
+     *
+     * @param name               the name to be given to the settings
+     * @param initialPIDSettings the PID settings to be added
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/com/spikes2212/dashboard/Namespace.java:193]: the initial values for the pid settings

File: src/main/java/com/spikes2212/dashboard/Namespace.java [hunk 7]
```diff
@@ -14,58 +13,186 @@
...
+     *
+     * @param key the key of the value
+     * @return the desired value
+     */
     boolean getBoolean(String key);
 
+    /**
+     * Adds a set of {@link PIDSettings} values to the namespace.
+     *
+     * @param name               the name to be given to the settings
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/com/spikes2212/dashboard/Namespace.java:192]: and the child namespace

File: src/main/java/com/spikes2212/dashboard/Namespace.java [hunk 8]
```diff
@@ -14,58 +15,189 @@
  */
 public interface Namespace {
 
+    /**
+     * Adds a double value to the namespace.
+     * Can only be changed via {@link Shuffleboard}.
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/com/spikes2212/dashboard/Namespace.java:20]: the, or another {@NetworkTable} UI.

File: src/main/java/com/spikes2212/dashboard/Namespace.java [hunk 9]
```diff
@@ -14,58 +15,189 @@
...
+    /**
+     * Adds a {@link ChildNamespace} to this namespace.
+     *
+     * @param name the name given to the child namespace's name
+     * @return the new {@link ChildNamespace} that is created
+     */
     ChildNamespace addChild(String name);
 
+    /**
+     * Adds a sendable data to the namespace.
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/com/spikes2212/dashboard/Namespace.java:57]: {@link Sendable} data

File: src/main/java/com/spikes2212/dashboard/Namespace.java [hunk 10]
```diff
@@ -14,58 +15,189 @@
...
+     * @param key          the key that will be given to the value
+     * @param runnable     the runnable value to be added
+     * @param runOnDisable whether the instant command should be executable when the robot is disabled
+     */
     default void putRunnable(String key, Runnable runnable, boolean runOnDisable) {
         this.putData(key, new InstantCommand(runnable).ignoringDisable(runOnDisable));
     }
 
+    /**
+     * Adds a runnable value that can run on disable to the namespace.
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/com/spikes2212/dashboard/Namespace.java:97]: add the {@InstantCommand} description

**Review by Yoel-Perman-Brilliant — CHANGES_REQUESTED** [review]

**Review by Spikes-2212 — APPROVED** [review]

**Review by Yoel-Perman-Brilliant — APPROVED** [review]

**Pull request merged**